docs/book/cookbook/aot-guide.md

```diff
@@ -297,36 +297,10 @@ the following files:
 
 Now we need to make the service manager use the AoT code.
 
-First, we'll create a delegate factory to decorate the DI injector with the AoT
-version. Decorating the injector ensures that your factories that utilize
-`Zend\Di\Container\AutowireFactory` will benefit from AoT as well.
-
-Create the file `src/AppAoT/src/InjectorDecoratorFactory.php` with the following
-contents:
-
-```php
-namespace AppAoT;
-
-use AppAoT\Generated\GeneratedInjector;
-use Interop\Container\ContainerInterface;
-use Zend\ServiceManager\Factory\DelegatorFactoryInterface;
-
-class InjectorDecoratorFactory implements DelegatorFactoryInterface
-{
-    public function __invoke(ContainerInterface $container, $name, callable $callback, array $options = null)
-    {
-        $injector = $callback();
-
-        if (class_exists(GeneratedInjector::class)) {
-            return new GeneratedInjector($injector);
-        }
-
-        return $injector;
-    }
-}
-```
-
-After this we need to add configuration to the `ConfigProvider` class we created
+First, we'll use a delegator `Zend\Di\GeneratedInjectorDelegator` to decorate
```

Comment:
I think we need to note that `GeneratedInjectorDelegator` is available since version X.Y.Z.

Reply:
Absolutely

Reply:
Docs are updated. If somebody has a more elegant phrasing or suggesstion, I'd appreciate to hear it.